hack/yaml-lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then 
+  yamllint --config-data "{extends: default, rules: {line-length: {level: warning, max: 120}}}" ./examples/ ./installer/
+else
+  docker run -t -v "$(pwd)":/workdir --env IS_CONTAINER='TRUE' --entrypoint sh quay.io/coreos/yamllint ./hack/yaml-lint.sh
```

Comment:
This is not an issue for Travis, but if we want to support devs running this locally, I think we want `docker run -t --rm ...` here.

Reply:
Done